src/components/small-card/small-card.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC, useState } from "react";
+import { ISmallCardProps } from "./small-card.interface";
+import { StylesWrapper } from "./small-card.styles";
+import { TransitionImage } from "../image";
+import { imagePlaceholder } from "@/shared/utils/image-placeholder";
+import { FavoriteButton } from "../favorite-button";
+import Link from "next/link";
+
+export const SmallCard: FC<ISmallCardProps> = ({
+  data,
+  session,
+  onLikeClick,
+}) => {
+  const [isLiked, setIsLiked] = useState<boolean>(false);
+
+  const handleGetLevel = (level: number) => {
+    if (level === 1) return "Low";
+    if (level === 2) return "Medium";
+    if (level === 3) return "High";
+  };
+
+  const handleLikeClicked = () => {
+    if (!session) {
+      onLikeClick();
+    } else {
+      setIsLiked(!isLiked);
+    }
+  };
+
+  if (!data) {
+    return null;
+  }
+
+  return (
+    <StylesWrapper>
+      <div className="card-image">
+        <Link href={`/plant/${data._id}`}>
+          <TransitionImage
+            image={data.images[0].src}
```

Comment:
Hi! Be aware that if images[0] does not have an entry, this will lead to uncaught error :) You can either set a default, check with an if statement, or use chain operator like so: `data?.images?.[0]?.src`

Reply:
🥔 Just to be safe, I did both. I updated to use the chain operator like so and also added default value to the image component if the image is not found / undefined